core/executor/src/lib.rs

```diff
@@ -115,12 +140,29 @@ impl Executor for AxonExecutor {
             };
 
             r.logs = backend.get_logs();
-            gas_use += r.gas_used;
+            gas += r.gas_used;
+            fee = fee.checked_add(r.fee_cost).unwrap_or(U256::max_value());
```

Comment:
if fee overflow, then fee != gas * gas_price, this will lead to some audit or accounting problem. Return error directly instead of `U256::max_value()`?

Reply:
The U256 overflow is impossible. The `exec()` function should not return an error, so use max value only eliminate theoretical overflow possibilities.